zipline/finance/performance/period.py

```diff
@@ -212,6 +245,27 @@ def rollover(self):
             else:
                 del self._payout_last_sale_prices[asset]
 
+    def subdivide_period(self, capital_change):
+        self.calculate_performance()
+
+        # We are segmenting a previous subperiod that ran up until this point.
+        # Those performance stats are the total performance stats at this point
+        self._prev_subperiod_returns = self.total_returns
```

Comment:
These values operated on at the same time. The copy, especially, would benefit from having a single handle to the group of values which are affected by the period subdivisions. e.g. making a `namedtuple` like the `PeriodStats` used elsewhere. I do not have an immediate suggestion for what the name of that tuple should be.

(Not a merge blocker, but may help longterm.)